src/niclips/figures/multi_view.py

```diff
@@ -21,21 +23,22 @@ def multi_view_frame(
     axes: list[int],
     vmin: float | None = None,
     vmax: float | None = None,
-    overlay: nib.Nifti1Image | None = None,
+    overlay: list[nib.Nifti1Image] = [],
```

Comment:
Might be better to support a single image or list of images, to avoid breaking compatibility?

Reply:
Updated this in 47a3381 - if a single image is passed, it getes converted to a list to make use of the updated functionality addressing lists. Functionally, will still perform the same.